docs/pt-br/essentials/named-routes.md

```diff
@@ -0,0 +1,31 @@
+# Rotas nomeadas
+
+Às vezes, é mais conveniente identificar uma rota com um nome, especialmente quando se liga a uma rota ou realiza navegações. Você pode dar a uma rota um nome nas opções `routes` ao criar a instância do roteador:
+
+``` js
+const router = new VueRouter({
+  routes: [
+    {
+      path: '/user/:userId',
+      name: 'user',
+      component: User
+    }
+  ]
+})
+```
+
+Para vincular a uma rota nomeada, você pode passar um objeto para o `router-link` componentes `to` prop:
```

Comment:
I believe is better change  **um objeto para o `router-link` componentes `to` prop:** to **um objeto para o componente `router-link` através da propriedade `to`**